src/main/java/uk/gov/digital/ho/proving/income/audit/AuditClient.java

```diff
@@ -46,7 +45,7 @@
     private final ObjectMapper mapper;
 
     public AuditClient(Clock clock,
-                       RestTemplate restTemplate,
+                       @Qualifier("createAuditRestTemplate")RestTemplate restTemplate,
```

Comment:
Check the spacing between the annotation and the class name here - you might want to run the IntelliJ autoformatter across your code.